src/qt/languages/fi-FI.po

```diff
@@ -143,16 +143,16 @@ msgid "&Average"
 msgstr "&Keskiarvo"
 
 msgid "CGA/PCjr/Tandy/E&GA/(S)VGA overscan"
-msgstr "CGA/PCjr/Tandy/E&GA/(S)VGA &yliskannaus"
+msgstr "CGA/PCjr/Tandy/E&GA/(S)VGA-&yliskannaus"
 
 msgid "Change contrast for &monochrome display"
 msgstr "&Muuta harmaavärinäytön kontrastia"
 
 msgid "&Media"
-msgstr "&Media"
+msgstr "&Levyt"
```

Comment:
Not every media is a disk

Reply:
Yes, the length is the reason for shortening to "Media" as in "Tallennusmedia". Many other translations are also affected by not having enough space in the Win32 UI where the translation was originally done. The UI has had some rework after that though and probably now fits longer words.